HW16/tests/test_hw16.py

```diff
@@ -0,0 +1,25 @@
+from HW16.pageobjects.MainPage import MainPage
+
+
+def test_login(browser):
+    MainPage(browser)\
```

Comment:
test should contain separate steps and asserts to maintain tests and reports readability

Reply:
reworked that part. 
Thanks